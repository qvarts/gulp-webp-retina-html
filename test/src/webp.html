<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Webp images</title>
</head>
<body>
<img id="testId" class="test-class" src="../img/test.png" alt="test">
<img class="test-class" src="../img/test.png" alt="test" />
<img src="../img/test.png" alt="test" class="test-class" />
<img class="test-class" src="../img/test.jpg" alt="test" />
</body>
</html>