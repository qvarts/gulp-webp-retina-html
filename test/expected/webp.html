<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Webp images</title>
</head>
<body>
<picture>
<source srcset="../img/test.webp" type="image/webp">
<img id="testId" class="test-class" src="../img/test.png" alt="test">
</picture>
<picture>
<source srcset="../img/test.webp" type="image/webp">
<img class="test-class" src="../img/test.png" alt="test" />
</picture>
<picture>
<source srcset="../img/test.webp" type="image/webp">
<img src="../img/test.png" alt="test" class="test-class" />
</picture>
<picture>
<source srcset="../img/test.webp" type="image/webp">
<img class="test-class" src="../img/test.jpg" alt="test" />
</picture>
</body>
</html>